garrysmod/lua/includes/util/color.lua

```diff
@@ -36,16 +36,27 @@ function IsColor( obj )
 end
 
 
+
 --[[---------------------------------------------------------
-	Returns color as a string
+	Returns color as a string ( unlinks necessity of
+	using tostring for functionality )
 -----------------------------------------------------------]]
-function COLOR:__tostring()
+function COLOR:ToString()
 	
 	return string.format( "%d %d %d %d", self.r, self.g, self.b, self.a )
 	
 end
 
 --[[---------------------------------------------------------
+	Returns color as a string
+-----------------------------------------------------------]]
+function COLOR:__tostring()

```

Comment:
I want to allow developers to alter __tostring to output something like Color( r, g, b, a ) or Colour, but ToString lets the serialized data be used in key-value, etc...

Falco suggested a better name, instead of ToString something along the lines of Show and I am swinging that route... ToString seems too vague and too related to __tostring or tostring( x ) so Show( ) or Serialize( ) or some other name may be better suited and I don't mind altering it for this update ( because it paves the way for all of the __tostrings to be created / modified ).